src/erlcloud_cur.erl

```diff
@@ -0,0 +1,133 @@
+-module(erlcloud_cur).
+
+
+%% AWS Cost and Usage Report API implementation
+%% https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/api-reference.html
+
+-include("erlcloud_aws.hrl").
+
+%% Initialization API
+-export([new/2, new/3, new/4, new/5]).
```

Comment:
@motobob do you think it's ok to remove `configure` functions and ban using of process dictionary for this new module?

re https://github.com/erlcloud/erlcloud/issues/204